docs/gatsby-node.js

```diff
@@ -30,7 +24,7 @@ exports.onCreateNode = ({ node, getNode, actions, reporter }) => {
             : node.absolutePath
         }", every directory in "src/documentation" should have one`,
       );
-      metaFileDataMap.set(url, {});
+      await cache.set(url, {});
```

Comment:
Would it make sense to make the cache key more specific? Like about it being the meta file? I don't know if there are other things that use similar keys or might be in the future. If so, it might help, but not sure it's a problem or not.

Reply:
I was thinking about this as well, but docs don't mention which cache keys I should avoid, so for now I'll pretend as if each plugin has namespaced cache, including the root `gatsby-node.js`, and that I don't have to worry about overriding anything.

Reply:
The cache is stored under `default-site-plugin`, which means all data from `gatsby-*` (node, config, etc.). The only cache I see there now is from this action, so it doesn't seem like a current problem. Just didn't know if it wouldn't be in the future, but probably not.